prime-dotnet-webapi/Configuration/AccessTerms/LicenseClassClauseMappingConfiguration.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Prime.Models;
+
+namespace Prime.Configuration
+{
+    public class LicenseClassClauseMappingConfiguration : SeededTable<LicenseClassClauseMapping>
+    {
+        public override ICollection<LicenseClassClauseMapping> SeedData
+        {
+            get
+            {
+                return new[] {
+                    new LicenseClassClauseMapping { LicenseCode = 1, OrganizatonTypeCode = 2, LicenseClassClauseId = 1, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new LicenseClassClauseMapping { LicenseCode = 1, OrganizatonTypeCode = 2, LicenseClassClauseId = 2, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new LicenseClassClauseMapping { LicenseCode = 1, OrganizatonTypeCode = 3, LicenseClassClauseId = 1, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new LicenseClassClauseMapping { LicenseCode = 1, OrganizatonTypeCode = 3, LicenseClassClauseId = 2, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE }
+                };
+            }
+        }
+
+        public override void Configure(EntityTypeBuilder<LicenseClassClauseMapping> builder)
+        {
+            builder.HasKey(m => new { m.LicenseCode, m.OrganizatonTypeCode, m.LicenseClassClauseId });
```

Comment:
the key should not include the license class clause id

Reply:
Yes it needs to be there to create a unique entry